packages/vm/src/evm/opcodes/functions.ts

```diff
@@ -1019,33 +1020,56 @@ export const handlers: Map<number, OpHandler> = new Map([
   [
     0xf6,
     async function (runState) {
-      const [commitUnpadded, yParity, r, s] = runState.stack.popN(4)
-      if (s > SECP256K1_ORDER_DIV_2) {
+      // eslint-disable-next-line prefer-const
+      let [authority, memOffset, memLength] = runState.stack.popN(3)
+
+      if (memLength > 128n) {
+        memLength = 128n
```

Comment:
can you use `BigInt(128)` instead of `128n` (we have been avoiding `n` usage for now)